### DISCUSSION TITLE
snake head outside the board

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#23] [Bug Reports]
If a snake is on the top row , and the snake doesn't respond in time.  The engine-cli will default to move up. On the next move, the engine-cli will send invalid request 
In the file, we can see:  
"board": {
		"height": 11,
		"width": 11,

So everything should have X & Y  between 0 and 10

But the engine send this part:
				"name": "Gamma",
				"health": 93,
				"body": [
					{
						"x": 1,
						"y": **11**
					},

[snakeHeadOutside.txt](https://github.com/BattlesnakeOfficial/rules/files/6286195/snakeHeadOutside.txt)

1. engine-cli should keep moving the snake in the same direction if snake timeout.
2. engine-cli shouldn't send outside coordinate.

The point 2 is may caused by the fact engine-cli send the snake info even after the snake got eliminated.

**Comment by bvanvugt** [comment]
Related to #33.

**Comment by bvanvugt** [comment]
Should be fixed in [Rules v1.0.18](https://github.com/BattlesnakeOfficial/rules/releases/tag/v1.0.18).
